### PR TITLE
Fix dougs st00f

### PR DESCRIPTION
**Pull request by RSNFreud**
make it better pl0x

### CODE REVIEW

**Comment by Arzte** [comment]
This isn't dougley's fault, I was the one to make the Readme to where it is now. On that note 👍

File: README.md [hunk 1]
```diff
@@ -8,9 +8,19 @@ npm i
 ```
 in the repository root
 
+You also need to install rethinkdb by downloading it and running it from command prompt
```
Comment (Arzte) [inline README.md]: You will also need ``[rethinkdb](https://www.rethinkdb.com/)``

File: README.md [hunk 2]
```diff
@@ -8,9 +8,19 @@ npm i
...
+
+Please remember to insert the uservoice details into the config as well as bot token and role ids for Admins/Mods.
```
Comment (Arzte) [inline README.md]: Before running the bot there is a few items to be replaced, they are as follows, replace the uservoice details with your info's, the discord bot token, and role ids (from the server that you are wanting to run the bot in, you need to make sure you have these roles added to your user or else you wouldn't be able to use the bot!).

File: README.md [hunk 3]
```diff
@@ -8,9 +8,19 @@ npm i
...
+
 # Run
 You can run the bot just as easliy as you can install it, just run
+
+```npm run-script dbcreate
+```
+
```
Comment (Arzte) [inline README.md]: Move this to the install section right under the mention for downloading and install rethinkdb

File: README.md [hunk 4]
```diff
@@ -8,9 +8,19 @@ npm i
...
 ```
 npm start
 ```
 in the repository root
+
+Make sure that rethinkdb is running!
```
Comment (Arzte) [inline README.md]: If you are getting a bunch of errors in console, make sure that rethinkdb is running

**Review by Arzte — CHANGES_REQUESTED** [review]
A few rewrites, and a couple of things that need to be moved.

**Comment by Dougley** [comment]
@SirDoctors How's this?

**Comment by Arzte** [comment]
even better, merged.

**Pull request merged**